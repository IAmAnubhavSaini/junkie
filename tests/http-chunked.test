#!/bin/sh

NAME="HTTP-chunked"
PCAPS="pcap/http/transfert_encoding_chunked.pcap"
filter() {
	# We keep only the GET because:
	# 1- it's enough to show that chunks are decoded properly
	# 2- proto for info are likely to change in the future (now dumper works as a per-packet basis which is not perfect)
	grep '^HTTP' | grep 'method=GET' | cut -d' ' -f 2-
}
OUTPUT="http-chunked.out"

. ./dump_test
