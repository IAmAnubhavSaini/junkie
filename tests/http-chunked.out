-- pcap/http/transfert_encoding_chunked.pcap --
head_len=906, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/s?hl=en&cp=1&gs_id=d&xhr=t&q=%3A&pf=p&sclient=psy-ab&site=&source=hp&pbx=1&oq=&aq=&aqi=&aql=&gs_sm=&gs_upl=&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=1&psi=Q-0vT-TaOcjT8gPZxKjWBg.1328540996352.1
head_len=910, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/s?hl=en&cp=3&gs_id=s&xhr=t&q=%3Ao%20&pf=p&sclient=psy-ab&site=&source=hp&pbx=1&oq=&aq=&aqi=&aql=&gs_sm=&gs_upl=&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=3&psi=Q-0vT-TaOcjT8gPZxKjWBg.1328540996352.1
head_len=932, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/csi?v=3&s=web&action=&ei=S-0vT-KlIZKS8gPN-ez6Dg&e=17259,30316,34903,35055,35211,35890,36045,36070,36121,36359,36361,36363,36370,36584&cp=false&imp=0&pf=1&pfa=n.3,ttfc.51,ttlc.0,cbt.63&pfm=n.3,ttfc.60,ttlc.0,cbt.63&imn=4&rt=prt.125,pprt.125,ol.125,jsrt.285,iml.126,kpr.636
head_len=921, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/s?hl=en&cp=5&gs_id=17&xhr=t&q=%3Ao%20ww&pf=p&sclient=psy-ab&source=hp&pbx=1&oq=&aq=&aqi=&aql=&gs_sm=&gs_upl=&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=5&psi=Q-0vT-TaOcjT8gPZxKjWBg.1328540996352.1
head_len=928, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/csi?v=3&s=web&action=&ei=Te0vT4CFCNPE8QOU-rmCDw&e=17259,30316,34903,35055,35211,35890,36045,36070,36121,36359,36361,36363,36370,36584&cp=false&imp=0&pfa=n.6,ttfc.61,ttlc.0,cbt.77&pfm=n.6,ttfc.88,ttlc.0,cbt.91&imn=2&rt=prt.109,pprt.109,ol.110,jsrt.258,iml.110,kpr.2202
head_len=875, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/csi?v=3&s=webhp&action=&e=17259,30316,34903,35055,35211,35890,36045,36070,36121,36359,36361,36363,36370,36584&ei=UO0vT_qbIMTB8QP-0Pz4BA&imc=2&imn=2&imp=0&rt=xjsls.33,prt.39,xjses.199,xjsee.240,ol.250,iml.39,xjs.255
head_len=919, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/s?hl=en&cp=2&gs_id=c&xhr=t&q=bl&pf=p&sclient=psy-ab&site=&source=hp&pbx=1&oq=&aq=&aqi=&aql=&gs_sm=&gs_upl=&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=2&psi=UO0vT_qbIMTB8QP-0Pz4BA.1328541008800.1
head_len=924, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/s?hl=en&cp=6&gs_id=11&xhr=t&q=blabla&pf=p&sclient=psy-ab&site=&source=hp&pbx=1&oq=&aq=&aqi=&aql=&gs_sm=&gs_upl=&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=4&psi=UO0vT_qbIMTB8QP-0Pz4BA.1328541008800.1
head_len=929, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/csi?v=3&s=web&action=&ei=VO0vT-zhEMSo8QOlne3oDg&e=17259,30316,34903,35055,35211,35890,36045,36070,36121,36359,36361,36363,36370,36584&cp=false&imp=0&pf=1&pfa=n.4,ttfc.67,ttlc.0,cbt.61&pfm=n.4,ttfc.131,ttlc.0,cbt.61&imn=3&rt=prt.98,pprt.98,ol.98,jsrt.376,iml.99,kpr.696
head_len=892, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.fr, user_agent=Mozilla/5.0 (X11; U; Linux x86_64; en-US; rv:1.9.1.16) Gecko/20110929 Iceweasel/3.5.16 (like Firefox/3.5.16), url=/search?q=blabla&hl=en&prmd=imvns&ei=VO0vT-zhEMSo8QOlne3oDg&sqi=2&start=10&sa=N&bav=on.2,or.r_gc.r_pw.,cf.osb&fp=40668b02a717aea1&biw=1919&bih=994&tch=1&ech=1&psi=UO0vT_qbIMTB8QP-0Pz4BA.1328541008800.3
